doc/source/cluster/vms/user-guides/launching-clusters/vsphere.md

```diff
@@ -0,0 +1,98 @@
+# Launching Ray Clusters on vSphere
+
+This guide details the steps needed to launch a Ray cluster in a vSphere environment.
+
+To start a vSphere Ray cluster, you will use the Ray cluster launcher with the VMware vSphere Automation SDK for Python.
+
+## Prepare the vSphere environment
+
+If you don't already have a vSphere deployment, you can learn more about it by reading the [vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/index.html). The following prerequisites are needed in order to create Ray clusters.
+* [A vSphere cluster](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-vcenter-esxi-management/GUID-F7818000-26E3-4E2A-93D2-FCDCE7114508.html) and [resource pools](https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-resource-management/GUID-60077B40-66FF-4625-934A-641703ED7601.html) to host VMs composing Ray Clusters.
+* A network port group (either for a standard switch or distributed switch) or an NSX segment. VMs connected to this network should be able to obtain IP address via DHCP.
```

Comment:
Are there pages we can link to for more about "network port group", "switch" and "NSX segment"?

Reply:
Yes. Added reference documentation pages for these elements.